### PR TITLE
Update WebSocket link to RFC6455

### PR DESCRIPTION
**Pull request by Hywan**

### CODE REVIEW

**Comment by Hywan** [comment]
ping?

File: README.md [hunk 1]
```diff
@@ -106,8 +106,8 @@ a request like
 
 followed by non-HTTP data.
 
-(See http://tools.ietf.org/html/draft-hixie-thewebsocketprotocol-75 for more
-information the Web Socket protocol.)
+(See [RFC6455](https://tools.ietf.org/html/rfc6455) for more information the Web
```
Comment (craig65535) [inline README.md]: might want to remove the space between Web and Socket, as above.

**Comment by Hywan** [comment]
@craig65535 Done :-).

**Comment by indutny** [comment]
Thank you, landed in 4f69be2221db0fa783b56621a45a2edab22cdc8e!

**Comment by Hywan** [comment]
Thanks!

**Pull request closed without merging**